lib/tools/adb-emu-commands.js

```diff
@@ -38,9 +38,15 @@ emuMethods.fingerprint = async function (fingerprintId, udid = undefined) {
   if (udid) {
     await this.setDeviceId(udid);
   }
+  this.resetTelnetAuthToken();
   await this.adbExec(['emu', 'finger', 'touch', fingerprintId]);
 };
 
+emuMethods.rotate = async function () {
```

Comment:
@mykola-mokhnach you're right I forgot to add 
```
if (udid) {
      await this.setDeviceId(udid);
}
```
yeah, the idea is to call it from `appium-android-driver`